playbook/app/pb_kits/playbook/pb_collapsible/collapsible_content.rb

```diff
@@ -12,6 +12,10 @@ def data
       def classname
         generate_classname("pb_collapsible_content_kit", "toggle-content", padding, separator: " ")
       end
+
+      def aria_content
+        {labelledby: "collapsible"}.merge! aria
+      end
```

Comment:
Really need to set defaults here incase they aren't added

Reply:
Think you should be in the clear here to use `merge!`.  My mistake was merging new values into the `aria` prop's memory location.

You're creating a new hash and then merging `aria` into it.  Which doesn't mutate `aria` prop's memory values, only accesses and copies them into the new hash you declared inline.